db/schema.rb

```diff
@@ -10,82 +10,82 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20200107163307) do
+ActiveRecord::Schema.define(version: 2020_02_06_201605) do
```

Comment:
why the format change, I like the new format, but why?

Reply:
We don't create the strings; they're a part of db:migrate.

Reply:
latter in the day I saw the same format in dashboard as well, so I retract my question.